### PR TITLE
🐛 Bug: app.js script banned

### LINKED ISSUE #177809: www.acquisa.de
### Issue URL (Annoyance)
 [https://www.acquisa.de/magazin/adblocker](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.acquisa.de%2Fmagazin%2Fadblocker)
Username: `@Spotlightforbugs 🐈‍⬛`

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard for Windows v7.17.1 (4709)
System version: | Win 11
Browser: | Edge
License type: | paid
AdGuard driver: | WFP
Stealth mode options: | Block WebRTC, <br>Block Flash, <br>Strip URLs from tracking parameters, <br>Hide your search queries, <br>Send Do-Not-Track header, <br>Hide your Referrer from third-parties, <br>Hide your User-Agent (Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.3), <br>Remove X-Client-Data header from HTTP requests, <br>Self-destructing third-party cookies (180), <br>Disable cache for third-party requests, <br>Block trackers, <br>Disable Windows Telemetry, <br>Turn off Advertising ID
DNS filtering: | server: `https://dns.cloudflare.com/dns-query`<br>filters: `https://filters.adtidy.org/windows/filters/15.txt`, `User rules`
Filters: | <b>Ad Blocking:</b><br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection, <br/>Legitimate URL Shortener<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances, <br/>AdGuard Cookie Notices, <br/>AdGuard Popups, <br/>AdGuard Mobile App Banners, <br/>AdGuard Other Annoyances, <br/>AdGuard Widgets, <br/>Adblock Warning Removal List<br/><br/><b>Security:</b><br/>Online Malicious URL Blocklist<br/><br/><b>Language-specific:</b><br/>AdGuard Turkish, <br/>AdGuard Russian, <br/>AdGuard Chinese, <br/>AdGuard German, <br/>AdGuard Japanese, <br/>AdGuard Spanish/Portuguese, <br/>EasyList Czech and Slovak, <br/>EasyList Italy, <br/>List-KR
Custom filters: | https://raw.githubusercontent.com/d3ward/toolz/master/src/d3host.adblock
Userscripts: | https://userscripts.adtidy.org/release/disable-amp/1.0/disable-amp.user.js<br>https://userscripts.adtidy.org/release/assistant/4.3/assistant.user.js<br>https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js<br>https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js<br>https://userscripts.adtidy.org/release/adguard-wot/1.0/wot.user.js

### PR DESCRIPTION
**Pull request by Coolhabit**
Fix: https://github.com/AdguardTeam/AdguardFilters/issues/177809

### CODE REVIEW

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt [hunk 1]
```diff
@@ -266,6 +266,7 @@ codesandbox.io,damota.me,pracazarohom.*,seduo.*#%#//scriptlet('remove-class', 'f
 !
 ! NOTE: Regular rules
 !
+||acquisa.de^*/app.js$script
```
Comment (Alex-302) [inline AnnoyancesFilter/Cookies/sections/cookies_specific.txt:269]: Too wide. This file is not only about subscription.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Comment by Coolhabit** [comment]
@Alex-302 @AdamWr changes made

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt [hunk 2]
```diff
@@ -269,6 +269,8 @@ codesandbox.io,damota.me,pracazarohom.*,seduo.*#%#//scriptlet('remove-class', 'f
 sportiva.shueisha.co.jp#%#//scriptlet('trusted-click-element', '#onetrust-accept-btn-handler')
 mypalmshop.de#%#//scriptlet('set-constant', 'eCookiePopupDialog', 'emptyObj')
 pca.st##.custom-cookie-banner
+acquisa.de#$#body { overflow: auto !important; }
+acquisa.de#$#.overlayer { display: none !important; }
```
Comment (AdamWr) [inline AnnoyancesFilter/Cookies/sections/cookies_specific.txt:273]: Please move these rules to - https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

**Review by AdamWr — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt (hunk 2 above)
Comment (Coolhabit) [reply AnnoyancesFilter/Cookies/sections/cookies_specific.txt]: @AdamWr done

**Review by AdamWr — APPROVED** [review]

**Pull request merged**